### PR TITLE
[FIX] fix a small bug in the curvature code

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

**Comment by arokem** [comment]
Is it possible to test this code with a test that fails without this fix?

File: AFQ/tests/test_segmentation.py [hunk 1]
```diff
@@ -60,6 +73,8 @@ def test_segment():
     npt.assert_equal(len(fiber_groups), 2)
     # Here's one of them:
     CST_R_sl = fiber_groups['CST_R']
+    import dipy.tracking.streamlinespeed as dps
```
Comment (arokem) [inline AFQ/tests/test_segmentation.py:76]: What is this import doing here?

Comment (36000) [reply AFQ/tests/test_segmentation.py:76]: I forgot to remove this!

**Comment by 36000** [comment]
I have no idea why this test is not working, as it seems to pass locally

**Comment by 36000** [comment]
it appears to be because I was using an older version of dipy

**Pull request merged**